src/base_model.py

```diff
@@ -0,0 +1,29 @@
+from peewee import Model, PostgresqlDatabase
+
+
+db: PostgresqlDatabase = PostgresqlDatabase(database="postgres",user="postgres",password="root",host="localhost")
```

Comment:
We should use environments variables here, because in the future we will have a production database with a better password 👍 An `.env` file is quite common.